content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -5,69 +5,77 @@ weight: $weight
-| TCP | 8444, 9080 | Internal | For web proxy <-> cnm_http/cm traffic |
-| TCP | 9081 | Internal, Active-Active | For Active-Active management |
-| TCP | 8070, 8071 | Internal, External | For metrics exported and managed by the web proxy |
+| TCP | 9081 | Internal, Active-Active | Active-Active management |
 | TCP | 9443 (Recommended), [8080](#turning-off-http-support) | Internal, External, Active-Active | REST API traffic, including cluster management and node bootstrap |
 | TCP | 10000-19999 | Internal, External, Active-Active | Database traffic |
-| TCP | 20000-29999 | Internal | Database shard traffic |
 | UDP | 53, 5353 | Internal, External | DNS/mDNS traffic |
 
-Connection sources are:
+### Ports for traffic between cluster nodes
 
-- Internal - The traffic is from other cluster nodes
-- External - The traffic is from client applications or external monitoring resources
-- Active-Active - The traffic is from clusters that host Active-Active databases
+These ports are used by Redis Software between cluster nodes:
 
-## Changing the management web UI port
+| Protocol | Port | Connection Source | Description |
+|------------|-----------------|-----------------|-----------------|
+| ICMP | * | Internal | Connectivity checking between nodes |
+| TCP | 1968 | Internal | Proxy traffic |
+| TCP | 3333-3341, 3343-3344, 36379, 36380 | Internal | Internode communication |
+| TCP | 20000-29999 | Internal | Database shard traffic |
 
-If for any reason you want to use a custom port for the RS Web UI
-instead of the default port (8443), you can change the port. Before you
-change the RS Web UI port, make sure that the new port is not in
-use by another process.
+### Ports for traffic within the cluster nodes
 
-{{< note >}}
-After you change the RS Web UI port, when you add a new node to the
-cluster you must connect to the web UI with the custom port number:
+These ports are used by Redis Software within each cluster node:
 
-`https://newnode.mycluster.example.com:`**`<nonstandard-port-number>`**
-{{% /note %}}
+| Protocol | Port | Connection Source | Description |
+|------------|-----------------|-----------------|-----------------|
+| TCP | 8002, 8004, 8006 | Internal | System health monitoring |
+| TCP | 8444, 9080 | Internal | Traffic betwen web proxy and cnm_http/cm |
+
+## Changing the management web UI port
+
+You can change the default port for the Redis Enterprise Software web UI (8443) to a custom port, as long as the new port is not in use by another process.
 
 To change the default port for the RS Web UI, on any node in the cluster run:
 
 ```sh
 rladmin cluster config cm_port <new-port>
 ```
 
+After changing the Redis Enterprise Software web UI port, you must connect any new node added to the cluster to the UI with the custom port number:
+`https://newnode.mycluster.example.com:`**`<nonstandard-port-number>`**
+
 ## Disabling HTTP support for API endpoints
```

Comment:
This belongs in the Security section, not here. I've created a ticket to address this.